layouts/partials/header.html

```diff
@@ -1,9 +1,13 @@
 <h1 class="site-title">
     <a href="{{ .Site.BaseURL | relLangURL }}">{{ .Site.Title }}</a>
 </h1>
-<nav>
-    {{ $currentPage := . }}
-    {{ range .Site.Menus.main }}
-    <a class="{{if or ($currentPage.IsMenuCurrent "main" .) ($currentPage.HasMenuCurrent "main" .) }} active{{end}}" href="{{ .URL | absLangURL }}" title="{{ .Title }}">{{ .Name }}</a>
-    {{ end }}
-</nav>
+{{ if .Site.Menus }}
```

Comment:
```suggestion
{{ if .Site.Menus.main }}
```

There can be other menus (like "about" for page in example site). So it should be limit to "main".